<launch>
  <node name="mls_upsampler_node" pkg="voxel_pkg" type="mls_upsampler_node" output="screen">
    <param name="input_pcd_file" value="$(find voxel_pkg)/data/251371071.pcd" />
    <param name="output_pcd_file" value="$(find voxel_pkg)/data/UPSAMPLED.pcd" />
    <!-- Set these values for setting up the up sampling amount -->
    <param name="m_upsampling_radius" value="0.025" />
    <param name="m_step_size" value="0.01" />
    <!-- Set the search radius according to your needs -->
    <param name="m_search_radius" value="0.3" />
    <param name="/use_sim_time" value="true" />
    <!-- Add new parameters -->
    <param name="input_cloud_topic" value="input_cloud" />
    <param name="upsampled_cloud_topic" value="upsampled_cloud" />
    <param name="input_cloud_viewer" value="Input Cloud Viewer" />
    <param name="upsampled_cloud_viewer" value="Upsampled Cloud Viewer" />
    <param name="point_size" value="1" />
    <param name="coordinate_system_size" value="10.0" />
    <param name="background_color_r" value="0" />
    <param name="background_color_g" value="0" />
    <param name="background_color_b" value="0" />
  </node>
</launch>
